### ISSUE TITLE
CRAN release still offers 0.18 binaries

### ISSUE THREAD

**Issue by cjyetman** [vincentarelbundock/countrycode#130]
as can be seen here https://cran.r-project.org/package=countrycode

when installing in R, it says...
```
  There is a binary version available but the source version is later:
            binary source needs_compilation
countrycode   0.18   0.19             FALSE

installing the source package ‘countrycode’
```

which gets the desired result (installing the latest version 0.19), but I wonder if there's something that could/should be done on this end to archive the 0.18 binary so it's not seen as a pseudo-option and directly available on the CRAN page?

**Comment by vincentarelbundock** [comment]
Is there a delay before binaries are built?

Teaching all day today, so can't really look into this now...

**Comment by cjyetman** [comment]
that may be the case... some of these have changed already since I first looked at it... https://cran.r-project.org/web/checks/check_results_countrycode.html

it also says [here](https://cran.r-project.org/web/packages/policies.html#Binary-packages)
"Binary packages are not accepted from maintainers: CRAN will only host binary packages prepared by those responsible for the binary areas. Their packages are made automatically by batch jobs and can take a day or two to appear on the CRAN master site (maybe longer to reach CRAN mirrors)."

Though that does not explicitly say that CRAN allows multiple versions of a package to be simultaneously offered in different forms, it seems that is what it does. I expected that the source version would not be released until it had passed compiling on all the test machines. I checked [another package](https://cran.r-project.org/web/packages/nlme/index.html) that was released yesterday, and the same type of thing is happening (currently).

I'll just close this... sorry for the false alarm.
